engineering/structure.md

```diff
@@ -0,0 +1,76 @@
+# Structure and responsibilities
+
+This page describes the structure and roles of our Engineering team.
+
+## Scope and responsibilities
+
+Below are is a summary of the major responsibilities for this team.
+
+### Oversee our technical strategy
+
+- Define our major objectives and priorities.
+- Make major architectural decisions about our cloud infrastructure.
+
+### Operate our cloud infrastructure
+
+- Perform daily operation of ongoing cloud services, addressing technical problems as-needed.
+- Resolve operational issues related to our cloud infrastructure.
+- Oversee our [Incident Response process](../projects/managed-hubs/incidents.md).
+
+### Develop our cloud infrastructure
+
+- Improve the reliability and uptime of our infrastructure
+- Improve the scalability of our infrastructure to more users and more communities
+- Minimize toil and automate processes as much as possible
+
+## Team structure
+
+This team operates similarly to a **Site Reliability Engineering** team.
+It focuses most development efforts on improving the cloud infrastructure that supports our partner communities, rather than developing new features and technology for the use-cases we support.
+It balances a combination of **operations** and **development** focused around our cloud infrastructure.
+
+Our **Technical Lead** and **Team Lead** define our technical priorities at any moment, and communicate these priorities to others on the team.
+They are expected to coordinate with other leads within 2i2c in order to ensure that our technical priorities reflect our organizational priorities for impact.
+Our **Project Manager** ensures that we have an efficient system for coordinating and planning our work, and **our engineering team** carries out most technical implementaitons and operations for 2i2c.
+
+(engineering:roles)=
+## Team roles
+
+Below is a list of each of the roles described above:
+
+```{glossary}
+
+Technical Lead
+
+  Responsible for defining our cloud strategy and technical priorities, and for carrying out major technical design and implementation for our infrastructure.
+
+  This role is currently filled by `@yuvipanda`.
+
+Team Lead
+
+  Responsible for overseeing our engineering team, ensuring that it is operating in an effective and equitable manner, and ensuring that each team member's work is aligned with their goals and interests.
+
+  This role is currently un-filled.
+
+Engineering Project Manager
+Project Manager
+Project Managers
```

Comment:
this is just so that both the singular and plural are treated as part of the same term, it's an annoying Sphinx glossary thing :-) but only the first item should actually show up on our rendered docs